prql-compiler/src/ast/item.rs

```diff
@@ -293,6 +293,11 @@ impl Display for Item {
                 UnOp::Not => write!(f, "not {}", expr.item)?,
             },
             Item::FuncCall(func_call) => {
+                match func_call.name.as_str() {
```

Comment:
When generating a group, we would have had something like: 
```
group [foo, bar]
aggregate [baz = sum foobar]
```

There were missing parentheses around aggregate, which led to invalid PRQL. With this code, we write an opening parenthesis before aggregate.

Another block of code writes the closing parenthesis, so that we get:
```
group [foo, bar] (
  aggregate [baz = sum foobar]
)
```

Reply:
Actually this is wrong, as groups do not only have aggregates, eg.:
```
group employee_id (
  sort month
  window rolling:12 (
    derive [trail_12_m_comp = sum paycheck]
  )
)
```

We should really put parentheses around the group's "content" and not around aggregate.

Reply:
Great!

And it's fine to leave a comment in the code for something that's not quite there yet!